templates/vhost/_directories.erb

```diff
@@ -493,6 +493,9 @@
     ProxyPassReverse <%= reverse_url %>
           <%- end -%>
         <%- end -%>
+        <%- if proxy['preserve_host'] -%>
```

Comment:
Should it deal with (in Puppet types) `Optional[Boolean]`? In other words: `undef` omits it, `true` is `On` and `false` is `Off`.

Reply:
According to the Apache docs, the default value is "Off". So if you don't put anything at all, it is off.

https://httpd.apache.org/docs/2.4/mod/mod_proxy.html#proxypreservehost

Because of that, I thought this would keep things more simple.

Reply:
What if you set `ProxyPreserveHost On` at the vhost level?

Reply:
That is a very good point. I'll update this tomorrow, thanks for catching it.